lib/provider/camunda/element-templates/Validator.js

```diff
@@ -110,6 +112,32 @@ function Validator() {
       }
     }
 
+    if (scopes) {
+      if (!isArray(scopes)) {
+        err = this._logError('invalid scopes, should be scopes=[]', template);
+      }
+      else {
+        var _this = this;
```

Comment:
Please declare at function root and call it `self` to comply with rest of code base.